suse/x86_64/suse-leap-42.3-JeOS/config.xml

```diff
@@ -63,6 +63,8 @@
         <package name="fontconfig"/>
         <package name="fonts-config"/>
         <package name="tar"/>
+        <package name="gptfdisk"/>
+        <package name="e2fsprogs"/>
```

Comment:
Aren't those only needed for vmx? I know it doesn't hurt here and I am totally fine with them. Just double checking that we are not using different tools from what is required in dracut-kiwi-oem-repart.

Reply:
gptfdisk provides gdisk which is needed to add a partition if the image is efi based. If it's just msdos table, fdisk would be enough

e2fsprogrs add mkfs.ext4 which is the filesystem I used in this example. If the data partition is something else e.g xfs other packages might be needed, but as the data partition has to have a filesystem there need to be the tool available to create that at boot time